QtAwesomeSample/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.16)
+project(QtAwesomeSample)
+
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core)
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core Widgets)
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+
+add_subdirectory(../QtAwesome QtAwesome)
```

Comment:
Ah, okey I see. What about to build QtAwesome without QtAwesomeSample?

Reply:
Usually, if I need QtAwesome, I have a Qt program already doing the find_package work.

Reply:
Sometimes you need to build library as a main project. Maybe ci/cd. Maybe you wanna add some tests

Reply:
Fixed here: ab15dd6